src/components/tx/TxData.js

```diff
@@ -367,7 +362,8 @@ class TxData extends React.Component {
     const renderOutputLink = idx => {
       if (idx in this.props.spentOutputs) {
         return (
-          <span>
+          // TODO for some reason, bold does not work with SF Pro
```

Comment:
@pedroferreira1 @tuliomir any idea why? I tried all options (bold, strong, 700, etc). I believe all text we have as bold on the website (that's currently working) uses Mona Sans font, not SF Pro.

Reply:
Based on our recent conversation, it seems the bold font files were available on the developer machines that implemented these changes ( and also on mine, as I couldn't reproduce the error ), but were not available in the repository for all users.

Adding those to the repository will probably solve this issue.

Reply:
Fixed by adding the font file (`src/assets/fonts/SFPRODISPLAYBOLD.OTF`) and definition on scss.